ddtrace/tracer/option_test.go

```diff
@@ -84,20 +94,25 @@ func TestTracerOptionsDefaults(t *testing.T) {
 	})
 
 	t.Run("other", func(t *testing.T) {
+		// Set a DD_ENV to ensure WithEnv overrides it.
+		os.Setenv("DD_ENV", "doodo")
```

Comment:
Can we use a better string? What's a `doodo`? Almost sounds like a 💩if it had another `o` at the end 🙂

Reply:
Oops. Meant to change this. 💩 💩 💩